metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/DepublishRecordIdService.java

```diff
@@ -239,12 +238,32 @@ public boolean canTriggerDepublication(MetisUser metisUser, String datasetId)
     } else {
```

Comment:
Only the else case is covered in your unit tests. Perhaps you can add a simple test case for the if case as well?

Reply:
You are right, I missed the `if` statement, I will add a new unit test